source/classification2.md

```diff
@@ -1387,9 +1544,11 @@ the $K$-NN here.
 <!--
 ## Predictor variable selection
 
-> **Note:** This section is not required reading for the remainder of the textbook. It is included for those readers 
-> interested in learning how irrelevant variables can influence the performance of a classifier, and how to
-> pick a subset of useful variables to include as predictors.
+```{note}
+This section is not required reading for the remainder of the textbook. It is included for those readers 
+interested in learning how irrelevant variables can influence the performance of a classifier, and how to
+pick a subset of useful variables to include as predictors.
```

Comment:
Should we uncomment this part so that it shows up in the textbook or is it not ready yet?

Reply:
not ready yet